src/main/scala/cromwell/engine/backend/jes/authentication/JesAuthentication.scala

```diff
@@ -20,7 +35,12 @@ object ProductionJesConnection {
  */
 trait JesAuthentication { self: JesConnection =>
 
-  def authenticated[A](f: JesInterface => A) = f(jesConnection)
+  def authenticatedAsCromwell[A](f: JesInterface => A) = f(jesCromwellConnection)
```

Comment:
because there is an `authenticatedAsUser` now so I thought it was clearer to change the old `authenticated` so you always know as what you're authenticated.